### PR TITLE
➖ unused utils

### PR DESCRIPTION
**Pull request by roiLeo**

### CODE REVIEW

**Comment by Jarsen136** [comment]
build failed
image

File: utils/vueNotification.ts
Comment (vikiival) [inline utils/vueNotification.ts:1]: Have we ever used this?

**Comment by vikiival** [comment]
- logged in https://github.com/kodadot/packages/issues/155

**Review by vikiival — APPROVED** [review]
Repo is getting fit for summer

**Review by Jarsen136 — APPROVED** [review]

**Comment by vikiival** [comment]
pay 15 usd

**Comment by roiLeo** [comment]
> Repo is getting fit for summer

yes-zyzz

**Pull request merged**